azure-pipelines/experiments/common4j.yml

```diff
@@ -0,0 +1,39 @@
+# azure-pipelines/pull-request-validation/common4j.yml
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: dev
+
+pool:
+  name: Hosted Windows 2019 with VS2019
+
+jobs:
+- job: Phase_1
+  displayName: Phase 1
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+    persistCredentials: True
+  - template: ../templates/steps/credscan-policheck.yml
+    parameters:
+      policheckCmdLineArgsDir: adal
```

Comment:
what is this adal reference here?

Reply:
Good catch, that shouldn't be adal. (this parameter is the folder to check with policheck)